LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2011-2017 AlphaSierraPapa for the SharpDevelop team
```

Comment:
:bulb: The .NET Foundation templates do not include a date here. Removing the date makes some things easier, but other things more complicated. I would recommend evaluating two options to determine if they offer the protections you desire:

1. Removing the date
2. Using a single date instead of a date range (specifically, the date at which the project was started)